File: tkpddigital/src/main/java/com/tokopedia/digital/cart/data/entity/requestbody/topcart/RequestBodyOtpSuccess.java
package com.tokopedia.digital.cart.data.entity.requestbody.topcart;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author anggaprasetiyo on 3/9/17.
 */

public class RequestBodyOtpSuccess {

    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("id")
    @Expose
    private String id;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

}

File: features/recharge/digital_checkout/src/main/java/com/tokopedia/digital_checkout/data/request/RequestBodyOtpSuccess.kt
package com.tokopedia.digital_checkout.data.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.tokopedia.common_digital.cart.data.entity.requestbody.RequestBodyIdentifier

/**
 * @author by jessica on 20/01/21
 */
data class RequestBodyOtpSuccess (
        @SerializedName("type")
        @Expose
        private var type: String,

        @SerializedName("id")
        @Expose
        private val id: String,

        @SerializedName("attributes")
        @Expose
        private val attributes: Attributes
) {
    data class Attributes(
            @SerializedName("ip_address")
            @Expose
            private var ipAddress: String,

            @SerializedName("user_agent")
            @Expose
            private val userAgent: String,

            @SerializedName("identifier")
            @Expose
            private val identifier: RequestBodyIdentifier
    )
}